packages/newspringchurchapp/src/prayer/data/mutations/savePrayer.js

```diff
@@ -1,7 +1,7 @@
 import gql from 'graphql-tag';
 
 export default gql`
-  mutation savePrayerRequest($nodeId: String!) {
+  mutation SavePrayer($nodeId: String!) {
```

Comment:
Reason behind this change? In Rock, the model name is PrayerRequest.

Reply:
we're Rock agnostic ;)